gridt/src/app/core/settings.service.ts

```diff
@@ -17,44 +17,46 @@ export interface ServerMessage {
   providedIn: "root"
 })
 export class SettingsService {
-  /*
-   * Subscribe to this observable to ready the API.
-   */
-  public URL = "https://api.gridt.org";
 
   constructor (
     private http: HttpClient,

```

Comment:
This dependency has become redundant because you have properly stored all api calls in the api service. This means you can remove reference of this here, as well as in your imports and in the tests, cleaning it up nicely.